lib/modules/versioning/semver/types.ts

```diff
@@ -0,0 +1,10 @@
+const SEMVER_X_RANGE = ['*', 'x', 'X', ''] as const;
+type SemVerXRangeType = typeof SEMVER_X_RANGE;
+export type SemVerXRange = SemVerXRangeType[number];
+
+/**
+ * https://docs.npmjs.com/cli/v6/using-npm/semver#x-ranges-12x-1x-12-
+ */
+export function isSemVerXRange(range: string): range is SemVerXRange {
```

Comment:
`types.ts` must only contain typescript types, so it's empty at compile time and never loaded at runtime. to move non types to `common` or ` util`